src/Rokt-Kit.js

```diff
@@ -494,6 +499,16 @@ function hashEventMessage(messageType, eventType, eventName) {
     );
 }
 
+function isEmpty(value) {
+    return (
+        value === null ||
+        value === undefined ||
+        (typeof value === 'object' && Object.keys(value).length === 0) ||
+        (typeof value === 'string' && value.length === 0) ||
+        (Array.isArray(value) && value.length === 0)
+    );
+}
+
```

Comment:
This is _probably_ okay, but I do see an `isEmpty` helper in our other kits like [Braze](https://github.com/mparticle-integrations/mparticle-javascript-integration-braze/blob/master-v5/src/BrazeKit-dev.js#L1175-L1177) which matches our TS version in [core](https://github.com/mParticle/mparticle-web-sdk/blob/master/src/utils.ts#L266-L267)

Reply:
Just realized that the reason is because the robot told you some wrong information when you did in fact use the code i referenced above.  I tested the robot's advice:

"The isEmpty function logic is flawed. If value is null or undefined, Object.keys(value) will throw an error before the null check can prevent it"

Null check is the first thing it does though...so when value is null or undefined it's working fine.  I'd keep it to what we had it as before, since we're using it in core/kits over the last few years.

<img width="607" height="208" alt="image" src="https://github.com/user-attachments/assets/4d921832-7a24-411e-8c1b-d75c23a731a5" />